### PR TITLE
Export feature - Implement export data as Excel function

### PR DESCRIPTION
**Pull request by Tiyop**
**Implement solution for #29 and #31** 

Changes impact files:
- app.py: for routing to download data url
- dahsboard.py: for linking button to download data url
- utils/__init__.py: for defining DataExport class and its functions. -> we can potentially put this class in another module.
- requirements.txt

Notes:
- In class DataExport there are two functions: get_file_as_bytes and get_file_as_bytes_openpyxl. The first one generates Excel file from scratch and the second uses a template ('data/templates/beges_template.xlsx)
-  Implemented solutions are slow (specially the one based on a template taking >30 sec to complete). **Need more perf optimization.**

### CODE REVIEW

File: app/src/apps/dashboard.py [hunk 1]
```diff
@@ -42,7 +45,13 @@
         dbc.Row(
             [
                 dbc.Col(
-                    dbc.Button("Exporter les données", color="primary", className="mr-1", block=True),
+                    [
+                        html.A(
```
Comment (antoan2) [inline app/src/apps/dashboard.py]: Interesting way to implement the redirection using flask.
Have you read this as a good practice ?
Seems lighter than having a hidden div as I am doing everywhere else :s

File: app/src/utils/__init__.py [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+class DataExport:
+    def __init__(self, selected_entity):
+        self.service = oc.get_entity_by_id(selected_entity)
+        self.load_data()
+
+    def load_data(self):
+        self.chorus_dt_df = ch.get_structure_data(self.service.code_chorus)
+        self.odrive_df = ov.get_structure_data(self.service.code_odrive)
+        self.osfi_df = oh.get_structure_data(self.service.code_osfi)
+        pass
```
Comment (antoan2) [inline app/src/utils/__init__.py:24]:
```suggestion

```

**Comment by Tiyop** [comment]
Rebased export-feature on master and did some cleaning

File: app/src/apps/dashboard.py (hunk 1 above)
Comment (Tiyop) [reply app/src/apps/dashboard.py]: I wanted to directly use the button href attribute as you would do a redirect in standard website but somehow it didn't synchro well with my callback. Found this turnaround on the web having the button inside an html component.

**Review by antoan2 — APPROVED** [review]
Seems ok on my side.

I would actually like to see in a next PR the `utils.DataExport(service)` been run on all the services of the organization chart as a preparation step and the route `/data/exportRaw` only serving the prepared file.

**Pull request merged**